conda_recipe/pytorch_deps_environment.yaml

```diff
@@ -0,0 +1,22 @@
+# This file may be used to create an environment using:
+# $ conda env create --name <env> --file conda_recipe/environment.yml
```

Comment:
- `pytorch_deps_environment.yaml`
- and I thought these files were all called `<something>.yml`